precise/iaf_psc_alpha_ps.cpp

```diff
@@ -578,36 +575,78 @@ nest::iaf_psc_alpha_ps::emit_instant_spike_( Time const& origin, const long lag,
 }
 
 double
-nest::iaf_psc_alpha_ps::bisectioning_( const double dt ) const
+nest::iaf_psc_alpha_ps::V_m_root_function_( double t_step ) const
```

Comment:
The function returns the difference of V_m and threshold, so I would call it `threshold_distance_`.